### DISCUSSION TITLE
Tailwind CSS not working (partly)

### DISCUSSION THREAD

**Discussion by f3bw** [tailwindlabs/tailwindcss#11033] [Help] [answered]
### Tailwind CSS not applying partly
Tailwind CSS does not apply to the components in the UI folder. Am I doing something wrong/missing something?

In the picture, I showed the path in the tailwind.config file and my folder layout. 

(It does work for my pages)

Could someone help me, please?

<img width="756" alt="Screenshot 2023-04-19 at 09 39 03" src="https://user-images.githubusercontent.com/61694102/233003902-2c315352-d513-4c7c-b4a9-b24ef18214d6.png">

**Comment by wongjn** [comment]
Consider checking:
- The class names are not built via string concatenation/interpolation – class names must appear as complete unbroken strings.
- There is not some other CSS on the page that may interfere with Tailwind classes, perhaps overriding Tailwind rules.
- Tailwind has run a compilation since you added the class names. Try restarting the dev/watch task if you have it running.
- The element(s) in question is visible on the page.

Otherwise, consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Reply by f3bw** [reply]
Thanks for your response, unfortunately, I could not fix the error.
This is a link to a PR where the explained unexpected behavior exists: [PR with unexpected behavior](-)

(edit: removed actual link for privacy reasons)

**Reply by wongjn** [reply]
Tailwind seems to be working as expected. I assume it is this UI element that is the focal point of your unexpected behavior?

![image](https://user-images.githubusercontent.com/11310624/233061375-06a19765-13ec-4a50-bb59-bb628914a1d6.png)

Could you describe in more detail about how your experience is different?

**Reply by f3bw** [reply]
Yes indeed, it does work in the pages directory. But when using the command 'yarn storybook' a window pops up with the same component (button) but without the styling. That is the problem I am having right now.

**Reply by wongjn** [reply, marked as answer]
Thank you for the clarification. It seems like CSS generated by Tailwind is not being loaded into the Storybook. Consider looking at adding the CSS generated by Tailwind into the Storybook to have the components styled as you expect. For example, in `.storybook/preview.ts`:

```diff
 import type { Preview } from "@storybook/react";
+import 'tailwindcss/tailwind.css';

 const preview: Preview = {
```

**Reply by f3bw** [reply]
Thank you so much for this comment. By adding the line '`import ../src/styles/globals.css'` to the top of the `.storybook/preview.ts` file the styling is working. This is the file where I added Tailwind to my PostCSS configuration
[Step 4 of the official documentation](https://tailwindcss.com/docs/installation/using-postcss).

Once again, thank you!

**Comment by priyankashah3107** [comment]
I am also facing the same problem, tailwind css is not working on my react project after trying, can anyone help me?

**Reply by HAPPY321SUV** [reply]
Same problem u help me code and code send
